packages/clerk-js/src/ui/customizables/index.ts

```diff
@@ -32,6 +32,18 @@ export const Image = makeCustomizable(sanitizeDomProps(makeResponsive(Primitives
 export const Alert = makeCustomizable(sanitizeDomProps(Primitives.Alert));
 export const AlertIcon = makeCustomizable(sanitizeDomProps(Primitives.AlertIcon));
 export const Input = makeCustomizable(sanitizeDomProps(Primitives.Input), { defaultDescriptor: descriptors.input });
+export const CheckboxInput = makeCustomizable(sanitizeDomProps(Primitives.Input), {
+  defaultDescriptor: descriptors.checkbox,
+  defaultProps: {
+    type: 'checkbox',
+  },
```

Comment:
Why did we add the `defaultProps` functionality here? I liked the concept we had the last time I reviewed. That is, we have a `CheckboxInput` element that is an `Input` with a descriptor and `type='checkbox'`. Similar to what we do with `Flex` and `Col`.

Reply:
Yeah I did that to avoid having another file, and because we already had `defaultStyles` and `defaultDescriptor` i thought that was a good addition.